examples/reference/pandas/hist.ipynb

```diff
@@ -6,14 +6,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import hvplot.pandas  # noqa"
+    "import hvplot.pandas  # noqa\n",
+    "\n",
+    "# hvplot.extension(\"matplotlib\")"
```

Comment:
Why do you use matplotlib?

Reply:
I don't use it. But I indicate how a user could change the backend.

Reply:
Its done to help the user. If you are not a super user of hvplot you 1. might not know that its possible or 2. remember how to do this.

The intent is to help the user and enable her to quickly use the notebook to explore and try out different options including changing the plotting backend.

Reply:
Yes but it's also a good idea to explain things in relevant sections. Following your argument, we could document anything anywhere, it seems to me this is a recipe for spaghetti documentation.

There are already explanations on how to get Matplotlib plots:
- on the home page
- in the getting started guide
- in a dedicated guide to plotting with Matplotlib
- in a guide that explains how to switch between plotting backends

It seems to me that it's enough.

At some point I'd like the reference gallery to also show Matplotlib and Plotly plots. This is what HoloViews does, except that to do that it has to duplicate the notebooks which makes it pretty annoying to make any sort of modification, since you have to modify three notebooks. I also makes it too easy for things to diverge. I wouldn't like to have this in hvPlot now, I prefer improving the default Bokeh reference pages first.

Reply:
I'f you where not tied to notebooks, you could develop a live app that could enable the user to select the data backed (pandas, xarray, duckdb etc), the plot, the plot backend.

This would also give you a lot of automated tests to help make hvplot and holoviews more robust.